src/crypto-providers/trezorCryptoProvider.ts

```diff
@@ -267,6 +313,23 @@ const TrezorCryptoProvider: () => Promise<CryptoProvider> = async () => {
     }
   }
 
+  const prepareTtl = (ttl?: number) => {
+    if (!ttl && !isFeatureSupportedForVersion(TrezorCryptoProviderFeature.ALLEGRA)) {
+      throw Error(Errors.TrezorOptionalTTLNotSupported)
+    }
+    return ttl && ttl.toString()
+  }
+
+  const prepareValidityIntervalStart = (validityIntervalStart?: any): string | undefined => {
```

Comment:
why any? Also I would prefer null over undefined as undefined can be a result of something unintentionally missing

Reply:
Fixed.